src/components/latestNews.js

```diff
@@ -1,24 +1,31 @@
 import React from "react";
-import { newsSelector} from "../selectors/newsSelector"
-import { BannerButton } from "./generics/bannerButton";
-import { DisplayBadge } from "./generics/displayBadge";
+import { newsSelector } from "../selectors/newsSelector";
+import { companySelector} from "../selectors/companySelector";
+import { statsSelector } from "../selectors/statsSelector";
+import { quoteSelector } from "../selectors/quoteSelector";
 import { DisplayWrapper } from "./generics/displayWrapper";
-import { PriceDisplay } from "./generics/priceDisplay";
 import { TitleHeader } from "./generics/titleHeader";
 import { Text } from "./generics/text";
-import { StatLine } from "./generics/statLine";
-
 import { useDispatch, useSelector } from "react-redux";
 
 export const LatestNews = () => {
   //Call necessary selectors for display data
   const dispatch = useDispatch();
-    const newsArticles = useSelector(newsSelector);
+    const newsInfo = useSelector(newsSelector);
+
+    const companyInfo = useSelector(companySelector);
+    const quoteInfo = useSelector(quoteSelector)
```

Comment:
I think you only need the newsInfo in this component, the other selectors aren't used here. Also, just be sure to remove all console logs before merging.